data/secscan_model/secscan_v4_model.py

```diff
@@ -190,6 +190,7 @@ def needs_reindexing_query(indexer_hash):
                 Manifest.select()
                 .join(ManifestSecurityStatus)
                 .where(
+                    ManifestSecurityStatus.index_status != IndexStatus.MANIFEST_UNSUPPORTED,
```

Comment:
do not attempt to rescan unsupported manifests

Reply:
Can we add a unit test for this?